src/random.rs

```diff
@@ -27,10 +28,12 @@ impl RandomURLConfig {
             return None;
         }
         Some(match self.type_ {
-            RandomURLType::PetName => petname::petname(
-                self.words.unwrap_or(2),
-                self.separator.as_deref().unwrap_or("-"),
-            ),
+            RandomURLType::PetName => petname::Petnames::large()
+                .generate_one(
+                    self.words.unwrap_or(2),
+                    self.separator.as_deref().unwrap_or("-"),
+                )
+                .expect("no names"),
```

Comment:
We can return from here and let the callee of this function handle the `None` case.

```suggestion
                ?,
```

Reply:
I thought about it first, but I think it makes sense to panic here. This should never happen. The crate comes with a word list, so if None is returned, something is seriously wrong with the crate.
On the other side, a panic is prod is never useful. Letting the callee handle the error is also not a bad idea.